systest/backup/common/utils.go

```diff
@@ -91,6 +101,179 @@ func CopyToLocalFs(t *testing.T) {
 	require.NoError(t, testutil.DockerCp(srcPath, copyBackupDir))
 }
 
+func AddSchema(t *testing.T, header http.Header, whichAlpha string) {
```

Comment:
decided against changing the name as we discussed?

Reply:
Resolved